File: Java 2022/maxEnd3.java
public class maxEnd3 {
    public int[] maxEnd3(int[] nums) {
        if (nums[0] >= nums[nums.length-1]) {
        nums[0] = nums[0];
        nums[1] = nums[0];
        nums[2] = nums[0];
        }
        else if (nums[0] <= nums[nums.length-1]) {
        nums[0] = nums[nums.length-1]; 
        nums[1] = nums[nums.length-1]; 
        nums[2] = nums[nums.length-1];
        }
        return new int[] { nums[0],nums[1],nums[2]};
    }
}

File: Python 2022/max_end3.py
def max_end3(nums):
  if (nums [0] < nums [len (nums) - 1]):
    for x in range (len (nums)):
      nums [x] = nums [len (nums) - 1]
  else:
    for x in range (len (nums)):
      nums [x] = nums [0]
  return nums
